src/assets/styles/other/_tn-styles.scss

```diff
@@ -1558,4 +1559,9 @@ $primary-dark: darken(map-get($md-primary, 500), 8%);
       visibility: hidden;
     }
   }
+
+  // focus state styles for mat-form-field
+  .mat-form-field.mat-focused:focus-within {
+    outline: 1px solid var(--alt-fg2);
```

Comment:
This should look the same as other field's focus states
![image](https://user-images.githubusercontent.com/24204846/156154251-e6b563d5-cf1f-46e4-a17a-2f1dc8a0161f.png)
![image](https://user-images.githubusercontent.com/24204846/156154283-57654da8-4c98-4d7f-9c15-a1a6126453c9.png)